Tests/AFError+AlamofireTests.swift

```diff
@@ -1,9 +1,25 @@
 //
 //  AFError+AlamofireTests.swift
-//  Alamofire
 //
-//  Created by Jon Shier on 8/28/16.
```

Comment:
Whoops. Do you have a script you run to add the header or anything I can use in the future to avoid this?